invenio_communities/communities/records/jsonschemas/communities/communities-v1.0.0.json

```diff
@@ -36,7 +36,15 @@
           ]
         },
         "record_policy": {
-          "description": "Community record policy.",
+          "description": "Deprecated field. Please use record_submission_policy instead.",
```

Comment:
why this deprecation and a new field?

Reply:
Originally, this PR had a rename of `record_policy` to `record_submission_policy`, which would make this change backwards-incompatible. 

Given that `record_policy` wasn't actually used for anything and that the new name `record_submission_policy` makes more sense, I suggested we go with the deprecation as an option. 

In general, some of the naming changes in this PR (settings labels, etc.) need an extra sign-off, since they're UI/UX changes for things that might be in help guides/docs.